app/formSchema/uiSchema.ts

```diff
@@ -654,10 +653,9 @@ const uiSchema = {
       fileTypes: EXCEL_FILE_EXTENSIONS,
     },
   },
-  evidenceOfConnectivitySpeeds: {
+  otherSupportingMaterials: {
     'ui:widget': 'FileWidget',
-    'ui:description':
-      'Evidence of connectivity speeds (e.g, screen captures of Speed Test results)',
+    'ui:description': 'Other supporting materials (if applicable)',
     'ui:options': {
       maxLength: MAX_LONG_INPUT_LENGTH,
```

Comment:
do we need an option to hide the "optional" keyword?

Reply:
any chance we could implement as a string label to render for optional? This way we'd be enforcing that all optional fields have some label even if it isn't `(optional)`

Reply:
> any chance we could implement as a string label to render for optional? This way we'd be enforcing that all optional fields have some label even if it isn't `(optional)`

That's well out of scope of this ticket as well as there are some fields iirc that are not required but they wanted optional hidden.

Reply:
> do we need an option to hide the "optional" keyword?

good catch, I did that and renamed `ignoreOptional` to `hideOptional` to make it a bit more clear